messages.c

```diff
@@ -109,6 +109,8 @@ static const char *error_strings[] =
 	"E: CA root certificate with Extended Key Usage\n", /* ERR_ROOT_CA_WITH_EKU */
 	"E: Extended Key Usage without any entries\n", /* ERR_EMPTY_EKU */
 	"E: Extended Key Usage lacks a required purpose\n", /* ERR_MISSING_EKU */
+	"E: Invalid length of domainComponent", /* ERR_DOMAINCOMPONENT_SIZE */
+	"E: Invalid length of unstructuredName", /* ERR_UNSTRUCTUREDNAME_SIZE */
```

Comment:
You should also update the for loop below. Maybe we should create a MAX_ERR or something that goes with the other defines which is probably easier to remember to update.

Reply:
Yeah, I've wondered about doing that before.  Addressed by https://github.com/kroeckx/x509lint/pull/25/commits/162ee1ffe0b1a6d84a84793e3cbc97b09014fb2c